moxie-protocol/schema.graphql

```diff
@@ -199,9 +199,12 @@ type SubjectToken @entity {
   reserveRatio: BigInt!
   "Total supply of the token"
   totalSupply: BigInt!
+  "total locked In staking"
+  totalStaked: BigInt!
```

Comment:
can we add totalLockedSubjectTokens in the summary.

Reply:
added